### PR TITLE
cmdline args: switch -j description from "CPUs" to "threads".

### PR DESCRIPTION
**Pull request by matthiaskrgr**
My system has a single cpu, with 2 cores and 2 threads each and cargo will launch 4 jobs automatically.
IMO "defaults to # of threads" is a bit clearer.

### CODE REVIEW

File: src/bin/bench.rs [hunk 1]
```diff
@@ -64,7 +64,7 @@ Options:
     -p SPEC, --package SPEC ...  Package to run benchmarks for
     --all                        Benchmark all packages in the workspace
     --exclude SPEC ...           Exclude packages from the benchmark
-    -j N, --jobs N               Number of parallel jobs, defaults to # of CPUs
+    -j N, --jobs N               Number of parallel jobs, defaults to # of threads
```
Comment (petrochenkov) [inline src/bin/bench.rs]: "hardware threads" maybe?
Just "threads" sounds even more ambiguous than "CPUs".

Comment (matthiaskrgr) [reply src/bin/bench.rs]: Sounds good to me, I'll wait a bit before updating in case there are more suggestions :)

**Comment by alexcrichton** [comment]
Given that we use a library called `num_cpus` to figure this out I'd personally be hesitant to do this in that most programmers have an intuition of what "number of CPUs" means despite many also having knowledge that precise usage of the term "CPU" can be pretty fuzzy. I don't think it's worth being explicit here about "hardware threads" and I'd naively think my system doesn't natively have threads but rather has CPUs.

All in all I'm not sure it's worth making such a change?

**Comment by lukaslueg** [comment]
`defaults to number of threads` has no meaning to me. "Threads" is something I set, "CPUs" is something I have. `defaults to number of CPUs` transports to me that the number of threads will be set to the number of CPUs I have.

**Pull request closed without merging**